specification/resources/resource-manager/Microsoft.Management/2017-11-01-preview/management.json

```diff
@@ -0,0 +1,673 @@
+        "parameters": [
+          {
+            "$ref": "#/parameters/GroupIdParameter"
+          },
+          {
+            "$ref": "#/parameters/ApiVersionParameter"
+          },
+          {
+            "$ref": "#/parameters/CreateGroupRequestParameter"
+          }
+        ],
+        "responses": {
+          "200": {
+            "description": "OK",
+            "schema": {
+              "$ref": "#/definitions/ManagementGroup"
+            }
+          },
+          "default": {
+            "description": "Error",
+            "schema": {
+              "$ref": "#/definitions/ErrorResponse"
+            }
+          }
+        },
+        "x-ms-examples": {
+          "PutManagementGroup": {
+            "$ref": "./examples/createOrUpdateManagementGroup.json"
+          }
+        }
+      },
+      "patch": {
+        "tags": [
+          "ManagementGroups"
+        ],
+        "operationId": "ManagementGroups_Update",
+        "description": "Update a management group.\n",
+        "parameters": [
+          {
+            "$ref": "#/parameters/GroupIdParameter"
+          },
+          {
+            "$ref": "#/parameters/ApiVersionParameter"
+          },
+          {
+            "$ref": "#/parameters/CreateGroupRequestParameter"
+          }
+        ],
+        "responses": {
+          "200": {
+            "description": "OK",
+            "schema": {
+              "$ref": "#/definitions/ManagementGroup"
+            }
+          },
+          "default": {
+            "description": "Error",
+            "schema": {
+              "$ref": "#/definitions/ErrorResponse"
+            }
+          }
+        },
+        "x-ms-examples": {
+          "PatchManagementGroup": {
+            "$ref": "./examples/updateManagementGroup.json"
+          }
+        }
+      },
+      "delete": {
+        "tags": [
+          "ManagementGroups"
+        ],
+        "operationId": "ManagementGroups_Delete",
+        "description": "Delete management group.\nIf a management group contains child resources, the request will fail.\n",
+        "parameters": [
+          {
+            "$ref": "#/parameters/GroupIdParameter"
+          },
+          {
+            "$ref": "#/parameters/ApiVersionParameter"
+          }
+        ],
+        "responses": {
+          "200": {
+            "description": "OK - management group deleted successfully"
+          },
+          "204": {
+            "description": "NoContent - management group does not exist"
+          },
+          "default": {
+            "description": "Error",
+            "schema": {
+              "$ref": "#/definitions/ErrorResponse"
+            }
+          }
+        },
+        "x-ms-examples": {
+          "DeleteManagementGroup": {
+            "$ref": "./examples/deleteManagementGroup.json"
+          }
+        }
+      }
+    },
+    "/providers/Microsoft.Management/managementGroups/{groupId}/subscriptions/{subscriptionId}": {
```

Comment:
Just trying to remember what we discussed - did we say we wont expose GET API through this path?

Reply:
Added GET API to make things consistent.